### PR TITLE
Support regular expression

### PR DESCRIPTION
**Pull request by sachaos**
# Changes

- :gift: Support regular expression
- Add a test for that.

### CODE REVIEW

**Review by julz — CHANGES_REQUESTED** [review]
This looks awesome, thanks so much! Couple nits:

File: config.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+			Regexp: reg,
+			Alias:  alias,
+		})
+	}
+
+	c.Rules = rules
+	return nil
+}
+
+func (c Config) findRule(path string) *Rule {
```
[the remark this answers is not in the document]
Comment (sachaos) [reply config.go]: Thank you for review.
I fixed at https://github.com/julz/importas/pull/3/commits/6439bcc50383850e9f7d7b93cd8fdacb05943ce8

File: config.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	Regexp *regexp.Regexp
+}
+
+func (r *Rule) aliasFor(path string) (string, error) {
+	str := r.Regexp.FindString(path)
+	if len(str) > 0 {
+		return r.Regexp.ReplaceAllString(str, r.Alias), nil
+	}
+
+	return "", fmt.Errorf("mismatch rule")
```
[the remark this answers is not in the document]
Comment (sachaos) [reply config.go]: I fixed at https://github.com/julz/importas/pull/3/commits/6c1a7d79d6cdcba473f564c22423248d1f241d34

**Comment by julz** [comment]
Thanks @sachaos this is really cool!

**Comment by sachaos** [comment]
@julz Thank you!

I've created PR to golangci-lint to enable to use this feature in that.
https://github.com/golangci/golangci-lint/pull/1899

**Pull request merged**